csrc/engine/rdma/rdma_context.cpp

```diff
@@ -104,9 +104,9 @@ int64_t RDMAContext::init(const std::string& dev_name, uint8_t ib_port, const st
     SLIME_LOG_DEBUG("Max Memory Region:" << device_attr.max_mr);
     SLIME_LOG_DEBUG("Max Memory Region Size:" << device_attr.max_mr_size);
     SLIME_LOG_DEBUG("Max QP:" << device_attr.max_qp);
-    SLIME_LOG_DEBUG("Max QP Working Request:" << device_attr.max_qp_wr);
-    SLIME_LOG_DEBUG("Max CQ:" << (int)device_attr.max_cq);
-    SLIME_LOG_DEBUG("Max CQ Element:" << (int)device_attr.max_cqe);
+    SLIME_LOG_DEBUG("Max QP Working Request: " << device_attr.max_qp_wr);
+    SLIME_LOG_DEBUG("Max CQ: " << (int)device_attr.max_cq);
```

Comment:
Some thinks of cast:

https://google.github.io/styleguide/cppguide.html#Casting